### PR TITLE
feat: Agenda List View

### PR DESCRIPTION
**Pull request by Ibwedagama**
#### Related
- [S27A1 - View List Agenda
](https://airtable.com/app7SdZInMN1pG6ZL/tblLy5A3qYF19fj1u/viwDUthwzD6mc5jka/recTTgwhlTGM1QRPw?blocks=hide)

#### Changes
- add `date-fns` package
- add date formatting lib/helper
- fetch agenda data from API
- show data to table
- remove scroll bar on `<body>` tag to prevent visual bug

#### Preview
Peek 2022-01-06 12-00

### Evidence

title: feat agenda list view
project: Portal Jabar
participants: @Ibwedagama @adzharamrullah @bangunbagustapa @yoslie @naufalihsank

### CODE REVIEW

File: src/repositories/agendaRepository.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import Repository from './Repository';
```
Comment (adzharamrullah) [inline src/repositories/agendaRepository.js:1]: wow so clean nice

**Review by adzharamrullah — APPROVED** [review]

File: src/repositories/agendaRepository.js (hunk 1 above)
Comment (Ibwedagama) [reply src/repositories/agendaRepository.js:1]: thanks @adzharamrullah 👍🏼

Comment (bangunbagustapa) [reply src/repositories/agendaRepository.js:1]: This pattern looks promising 😍

**Review by bangunbagustapa — APPROVED** [review]
Could you please also add padding / margin to the bottom of data table? Looks like it hit the bottom end.

File: src/repositories/agendaRepository.js (hunk 1 above)
Comment (Ibwedagama) [reply src/repositories/agendaRepository.js:1]: thinking about write an article about this @bangunbagustapa 🦖

**Comment by Ibwedagama** [comment]
> Could you please also add padding / margin to the bottom of data table? Looks like it hit the bottom end.

oops my bad, will fix it asap @bangunbagustapa

**Comment by Ibwedagama** [comment]
> > Could you please also add padding / margin to the bottom of data table? Looks like it hit the bottom end.
> 
> oops my bad, will fix it asap @bangunbagustapa

I add padding bottom to `Agenda` component `<main>` tag, I think my layout structure is not that great @bangunbagustapa 🤔

File: src/repositories/agendaRepository.js (hunk 1 above)
Comment (adzharamrullah) [reply src/repositories/agendaRepository.js:1]: i hope soon i get the article of that

**Review by yoslie — APPROVED** [review]

**Review by naufalihsank — APPROVED** [review]

**Pull request merged**